src/Forms/SearchableDropdownTrait.php

```diff
@@ -450,9 +457,16 @@ public function getSchemaDataDefaults(): array
 
     public function getSchemaStateDefaults(): array
     {
-        $data = parent::getSchemaStateDefaults();
-        $data = $this->updateDataForSchema($data);
-        return $data;
+        $state = [
```

Comment:
```suggestion
        $data = [
```
It's called $data in the ancestor SingleSelectField ancestor so should retain that for consistency. We also call the method updateDataForSchema, again it's using the word data not state

Reply:
It's called `$state` in `FormField` lol. It's also just a locally scoped variable so the name doesn't really matter. I'm inclined to keep it as `$state` because `getSchemaDataDefaults()` is a separate method which deals with data - whereas this one deals with state.